projects/04_cube_xr/CubeXr.cpp

```diff
@@ -275,8 +275,10 @@ void CubeXrApp::Render()
         float4x4 V = glm::lookAt(float3(0, 0, 0), float3(0, 0, 1), float3(0, 1, 0));
 
         if (IsXrEnabled()) {
-            P = GetXrComponent().GetProjectionMatrixForCurrentViewAndSetFrustumPlanes(0.001f, 10000.0f);
-            V = GetXrComponent().GetViewMatrixForCurrentView();
+            GetXrComponent().SetFrustumPlanes(0.001f, 10000.0f);
```

Comment:
I don't know why it is this specific value, `PPX_CAMERA_DEFAULT_{NEAR,FAR}_CLIP = {0.1f, 10000.0f}` which is different.

Reply:
Does it changes something to push the near plane a bit further?
I suppose this could be the same for both cameras. Maybe 10cm was considered too close when trying XR, so it was reduced to 1mm?
I'd assume it would be safe to set it to `0.1f` for both camera types.

Reply:
should be fine to use the same camera setup for now.

Reply:
Sorry I meant to say `0.001` for both.

Reply:
we could use the same near&far to calculate the projection matrix for both xr and non-xr cases.

Reply:
From my perspective, those values are arbitrary, so I could change it to something in this pr if we decided on some value or they can be updated later.

Reply:
I'd be in favor of using `PPX_CAMERA_DEFAULT_{NEAR,FAR}_CLIP` then

Reply:
Okay, I'll delete that line since `ppx::Camera` without modification initialize to `PPX_CAMERA_DEFAULT_{NEAR,FAR}_CLIP`